evadb/optimizer/optimizer_utils.py

```diff
@@ -283,8 +313,8 @@ def check_expr_validity_for_cache(expr: FunctionExpression):
     return (
         expr.name in CACHEABLE_FUNCTIONS
         and not expr.has_cache()
-        and len(expr.children) <= 1
-        and isinstance(expr.children[0], TupleValueExpression)
+        # and len(expr.children) <= 1
+        and isinstance(expr.children[1], TupleValueExpression)
```

Comment:
Why we need the second parameter to the function to be `TupleValueExpression`? What if the function only accepts one arguments?

Reply:
Fixed this.